aplt/runner.py

```diff
@@ -114,39 +116,181 @@ def remove_client(self, ws_client):
                 connectWS(self._factory, contextFactory=self._factory_context)
             return
 
-    def remove_processor(self, processor):
+    def remove_processor(self):
         """Remove a completed processor"""
-        self._processors.pop(processor, None)
+        self._processors -= 1
+
+
+class LoadRunner(object):
+        """Runs a bunch of scenarios for a load-test"""
+        def __init__(self, scenario_list):
+            """Initializes a LoadRunner
+
+            Takes a list of tuples indicating scenario to run, quantity,
+            stagger delay, and overall delay.
```

Comment:
nit: I wonder if this should be a dict instead of a tuple, only because then folks are explicit about what they're passing. It's less efficient, but might be more readable and understandable for someone unfamiliar with how things work.